OrbitGl/App.cpp

```diff
@@ -352,9 +351,6 @@ void OrbitApp::PostInit() {
   GCurrentTimeGraph->SetStringManager(string_manager_);
 
   if (HasTcpServer()) {
-    GTcpServer->AddCallback(Msg_MiniDump, [=](const Message& a_Msg) {
-      GOrbitApp->OnMiniDump(a_Msg);
-    });
```

Comment:
That's the code which is according to Pierric not needed anymore, right?

Reply:
It looks like we're going to replace this functionality with crashpad